{"name": "test_TC_SUR_04_max_character_input", "status": "failed", "statusDetails": {"message": "AssertionError: Form incorrectly accepted long input\nassert 'Survey Resp...successfully' not in 'Survey Resp...ur feedback.'\n  \n  'Survey Response: S...mitted successfully' is contained here:\n    Survey Response: Survey submitted successfully, Thank you for your feedback.", "trace": "self = <test_cases.test_survey_form.TestContactForm object at 0x10370fe90>\n\n>   ???\nE   AssertionError: Form incorrectly accepted long input\nE   assert 'Survey Resp...successfully' not in 'Survey Resp...ur feedback.'\nE     \nE     'Survey Response: S...mitted successfully' is contained here:\nE       Survey Response: Survey submitted successfully, Thank you for your feedback.\n\n/Users/tinopro14/Documents/TESTING SCRIPTS/tino-dev-testing/test_cases/test_survey_form.py:66: AssertionError"}, "start": 1749154880495, "stop": 1749154882542, "uuid": "d9ccdf07-fc4a-4a58-a034-103c5babce93", "historyId": "9901218d2aad33d987d641eb21f8a571", "testCaseId": "9901218d2aad33d987d641eb21f8a571", "fullName": "test_cases.test_survey_form.TestContactForm#test_TC_SUR_04_max_character_input", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_driver')"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_survey_form"}, {"name": "subSuite", "value": "TestContactForm"}, {"name": "host", "value": "tinos-MBP14"}, {"name": "thread", "value": "24217-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_survey_form"}]}